### ISSUE TITLE
montage.t failing for x86_64

### ISSUE THREAD

**Issue by pgajdos** [ImageMagick/ImageMagick#4864]
### ImageMagick version

7.1.0-24, 7.1.0-25

### Operating system

Linux

### Operating system, version and so on

openSUSE Tumbleweed

### Description

I get following test failure for x86_64 (not for i586, though):
```
$images->Set(bordercolor=>'blue', mattecolor=>'red')
$montage=$images->Montage( background=>'#696e7e',  tile=>'4x4', geometry=>'90x80+6+6>', frame=>'8x10',
  borderwidth=>'0', gravity=>'Center', shadow=>'True',background=>'gray' )
ReadImage()
Test 12, signatures do not match.
     Expected: c11aad0132d57f84ec42aad600f2c9f3fdb9ad5a64235cb08a296c0f1a509f6d
     Computed: 51b8db221299cea2bea84b11247bfa4b41a0cc3a9af27ff4c20f9c23ee7f2117
     Depth:    16
not ok 12
$images->Set(bordercolor=>'blue', mattecolor=>'red')
$montage=$images->Montage( background=>'#696e7e',  tile=>'4x4', geometry=>'90x80+6+6>', frame=>'8x10',
  borderwidth=>'0', gravity=>'Center', shadow=>'True', texture=>'granite:' )
ReadImage()
Test 13, signatures do not match.
     Expected: 292b115e08fcef4888ee3f386c6c3aff1c0ed008e34697435c1e4116a75149ed
     Computed: e2fe0b56decf6fd791813e99d7b0f40646a479589e7519d97e2f92969dd17a1e
     Depth:    16
not ok 13
```

### Steps to Reproduce

run PerlMagick via make test

**Comment by urban-warrior** [comment]
Its possible you don't have one of the required dependencies on your x86_64 host.  Can you post the t/test_12_out.miff and t/test_13_out.miff images here.  We'll compare them to the expected results and see if we can identify the issue.

**Comment by pgajdos** [comment]
Thanks, here you are:

[testresults.zip](https://github.com/ImageMagick/ImageMagick/files/8108850/testresults.zip)

(attaching MIFF is not supported?)

**Comment by pgajdos** [comment]
By the way, I have not seen it for 7.1.0-23.

**Comment by urban-warrior** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in the GIT main branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ https://imagemagick.org/download/beta/ by sometime tomorrow.

**Comment by pgajdos** [comment]
Confirming this is fixed with 7.1.0-26. Thanks!
